vcpkg-artifacts/session.ts

```diff
@@ -100,9 +97,6 @@ export class Session {
 
   /** register installer functions here */
   private installers = new Map<string, InstallerTool>([
-    ['nuget', installNuGet],
-    ['unzip', installUnZip],
-    ['untar', installUnTar],
```

Comment:
I'm pretty sure you still need these, they just become:

```suggestion
    ['nuget', installVcpkgExtract],
    ['unzip', installVcpkgExtract],
    ['untar', installVcpkgExtract],
```